From: Fabrice Bauzac <noon@mykolab.com>
Date: Thu, 29 Apr 2021 00:04:18 +0200
Subject: Don't start hp-systray in GNOME

Don't start hp-systray when there is no systray.  GNOME does not have
a systray anymore.  (Closes: #793675)
---
 hplip-systray.desktop.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hplip-systray.desktop.in b/hplip-systray.desktop.in
index 870dec6..12cf6eb 100644
--- a/hplip-systray.desktop.in
+++ b/hplip-systray.desktop.in
@@ -10,3 +10,4 @@ Terminal=false
 Categories=Application;Utility;
 X-KDE-StartupNotify=false
 StartupNotify=false
+NotShowIn=GNOME;
